CHANGELOG.rst

```diff
@@ -232,6 +232,11 @@ time or change existing behaviors in order to make them less surprising/more use
 * ``yield``-based tests are considered deprecated and will be removed in pytest-4.0.
   Thanks `@nicoddemus`_ for the PR.
 
+* ``[pytest]`` sections in ``setup.cfg`` files should now be named ``[tool:pytest]``
+  to avoid conflicts with other distutils commands (see `#567`_). ``[pytest]`` in
+  ``pytest.ini`` or ``tox.ini`` files are supported and unchanged.
```

Comment:
nitpick (feel free to ignore): "`[pytest]` *sections* in ...." or "*is* supported"